.github/workflows/build.yml

```diff
@@ -57,7 +57,7 @@ jobs:
       - name: Set up environment
         uses: ./.github/actions/setup
       - name: Build Web
-        run: ./bin/package-browser
+        run: yarn build:browser && yarn build:sync-server
```

Comment:
Hmm, is it required here? Below I see we are only uploading the desktop-client package, and the sync-server is built and uploaded separately

Reply:
good point, removed